@kgerheiser should grib_utils build with the current develop branch of ip?

It is failing to find a function. Is that because of the merge with ip2?

```
/usr/bin/ld: CMakeFiles/copygb.dir/copygb.F.o: in function `intgrib1_':
/home/runner/work/NCEPLIBS-grib_util/NCEPLIBS-grib_util/grib_utils/src/copygb/copygb.F:1222: undefined reference to `ipolates_'


```

--- a/.github/workflows/develop_test.yml
+++ b/.github/workflows/develop_test.yml
@@ -90,7 +90,7 @@ jobs:
       with:
         repository: NOAA-EMC/NCEPLIBS-ip
         path: ip
-        ref: develop
+        ref: v3.3.3
 
     - name: build-ip
       run: |

